lib/features.json

```diff
@@ -246,9 +246,14 @@
   },
   {
     "name": "selectingTradeMovesItems",
-    "description": "Selecting a trade automatically puts the required items into trading slots.",
+    "description": "selecting a trade automatically puts the required items into trading slots",
     "versions": ["1.14", "1.16.5"]
   },
+  {
+    "name": "resourcePackUsesHash",
```

Comment:
do you use this feature anywhere since you added it?

Reply:
L19 of reaource_pack.js in this branch